orchagent/fdborch.cpp

```diff
@@ -232,13 +232,14 @@ void FdbOrch::update(sai_fdb_event_t        type,
                            update.entry.mac.to_string().c_str(),
                            vlanName.c_str(), update.port.m_alias.c_str());
 
-            for (const auto& itr : m_entries)
+            for (auto itr = m_entries.begin(); itr != m_entries.end();)
```

Comment:
Looks good to me.
Just a suggestion, can we just update the for loop to the below, instead of increment iterator at multiple places?
```
for (auto itr = m_entries.begin(); itr != m_entries.end(); itr = next_item)
{
    auto next_item = std::next(itr, 1);
    .....
    ....
}

Reply:
Avoided multiple iterator increment. Updated patch. Pls check.